code/__HELPERS/screentips.dm

```diff
@@ -0,0 +1,33 @@
+#define HINT_ICON_FILE 'icons/ui_icons/screentips/cursor_hints.dmi'
+
+// Generate intent icons
```

Comment:
Change this instead to `/// A description of what this list is`, including the information that its a mapping of icon state to icon (helpfully even what those states are?).